<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2023-01-05T00:05:13" hostname="pc" time="0.015">
  <properties/>
  <testcase name="givenAValidInactivateCommand_whenCallsUpdateCategory_shouldReturnInactiveCategoryId()" classname="com.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.003"/>
  <testcase name="givenAValidCommand_whenCallsUpdateCategory_shouldReturnCategoryId()" classname="com.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.003"/>
  <testcase name="givenAValidComand_whenGatewayTrhowsRandomException_shouldReturnAException()" classname="com.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.003"/>
  <testcase name="givenACommandWithInvalidID_whenCallsUpdateCategory_thenShouldReturnNotFoundException()" classname="com.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.002"/>
  <testcase name="givenAInvalidName_whenCallsUpdateCategory_thenShouldReturnDomainException()" classname="com.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
